src/app/_components/most-popular-section/most-popular.tsx

```
@@ -0,0 +1,63 @@
+                ? "bg-maroon-600 text-white"
+                : "bg-gray-100 text-gray-700"
+            }`}
+          >
+            {o.name}
+          </button>
+        ))}
+      </div>
+
+      {/* Content */}
+      {data ? (
+        <div className="border rounded-lg p-4 bg-white shadow">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)